ooniapi/services/ooniprobe/src/ooniprobe/routers/v1/probe_services.py

```diff
@@ -0,0 +1,126 @@
+import logging
+from datetime import datetime, timezone, timedelta
+import time
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, Response
+
+from ...common.dependencies import get_settings
+from ...common.routers import BaseModel
+from ...common.auth import create_jwt, decode_jwt, jwt
+from ...common.config import Settings
+from ...common.utils import setnocacheresponse
+
+router = APIRouter(prefix="/v1")
+
+log = logging.getLogger(__name__)
+
+class ProbeLogin(BaseModel):
+    # Allow None username and password 
+    # to deliver informational 401 error when they're missing
+    username : str | None = None
```

Comment:
nitpick: Can you lint and reformat the code using `black` so that the spacing of the type hints is consistent with the rest of the codebase?